vizro-core/docs/pages/user-guides/actions.md

```diff
@@ -33,75 +33,73 @@ To enable downloading data, you can add the [`export_data`][vizro.actions.export
+                        color="species",
+                        size="petal_length",
+                    ),
                 ),
                 vm.Button(
                     text="Export data",
-                    actions=[
-                        vm.Action(
-                            function=export_data()
-                        ),
-                    ],
+                    actions=[vm.Action(function=export_data())],
                 ),
-            ],
+            ]
         )
 
         dashboard = vm.Dashboard(pages=[page])
-
         Vizro().build(dashboard).run()
         ```
 
+        1. We use a [`Flex`][vizro.models.Flex] layout to make sure the `Graph` and `Button` only occupy as much space as they need, rather than being distributed evenly.
+
     === "app.yaml"
 
         ```yaml
         # Still requires a .py to add data to the data manager and parse YAML configuration
         # See yaml_version example
         pages:
           - components:
-              - type: graph
-                figure:
+              - figure:
                   _target_: scatter
-                  data_frame: iris
-                  color: sepal_width
-                  x: petal_length
+                  x: sepal_width
                   y: sepal_length
-              - type: graph
-                figure:
-                  _target_: histogram
-                  data_frame: iris
                   color: species
-                  x: petal_length
+                  size: petal_length
+                  data_frame: iris
+                type: graph
               - type: button
                 text: Export data
-                id: export_data_button
+                id: export_data
                 actions:
                   - function:
                       _target_: export_data
-            title: Exporting
+            layout:
+              type: flex
+            title: My first page
         ```
 
     === "Result"
 
-        [![Graph]][graph]
+        [![ExportData]][exportdata]
 
 !!! note
 
-    Note that exported data only reflects the original dataset and any native data modifications defined with [`vm.Filter`](filters.md), [`vm.Parameter`](data.md/#parametrize-data-loading) or [`filter_interaction`](actions.md/#filter-data-by-clicking-on-chart) action. Filters from the chart itself, such as ag-grid filters, are not included, and neither are other chart modifications, nor any data transformations in custom charts.
+    Note that exported data only reflects the original dataset and any native data modifications defined with [`vm.Filter`](filters.md), [`vm.Parameter`](data.md/#parametrize-data-loading) or [`filter_interaction`](actions.md/#cross-filtering) action. Filters from the chart itself, such as ag-grid filters, are not included, and neither are other chart modifications, nor any data transformations in custom charts.
 
-### Filter data by clicking on chart
+### Cross-filtering
```

Comment:
This is a great improvement 🙌  Though I can't wait until we get rid of the whole thing and replace with the new scheme and then it'll be hugely better.